client/data/help/use-help-search-query.ts

```diff
@@ -23,6 +23,7 @@ const fetchArticlesAPI = ( search: string, locale: string, sectionName: string )
 	const queryString = buildQueryString( {
 		query: search,
 		locale,
```

Comment:
Would it be possible to avoid passing the locale param twice - `locale` and `_locale`?

Reply:
We'd have to update the backend to support this.

Reply:
It'd make sense to use a single param, if it's expected for `locale` and `_locale` to always share the same value, so I'd suggested to adjust the back-end, prior to merging this PR.